creating fastelane changelog (#3119)
TeamNewPipe/NewPipe: fastlane/metadata/android/en-US/changelogs/840.txt

New
• Language selector (#2921):
  ◦ Select your wanted language from Settings
  ◦ Applied after app restart

• Send to Kodi button (#2310):
  ◦ Displayed in player collapsible menu

• Copy comments via long press (#3053)

Improved

> • ReCaptcha solving gets correctly saved in the obtained cookie #3035

Reviewer: ```suggestion
• Fix crash with ReCaptchas and correctly store resolution cookie #3035
```

• Remove dot-menu in favour of drawer and hide history button when "Watch history" is not enabled in settings #2960

> • Ask for display over other apps permission in settings correctly on Android 6 and later #3004

Reviewer: ```suggestion
• Correctly request permission to display over other apps  in settings on Android 6 and later #3004
```

• Rename local playlist by long-clicking in BookmarkFragment #2954
• Various PeerTube improvements #3044:
  ◦ Display full description instead of truncated 250 characters
  ◦ Handle age limit
  ◦ Thumbnail in full quality

>   ◦ Recenty added feed fixed, was off as much as you were off from GMT

Reviewer 2: This makes sense to me because I was doing the PR, but I'm pretty sure otherwise I would not have understand this sentence.

The time, "X hours ago", was retrieved from the api in gmt time, and let as is, so if you don't live at gmt time, the time was wrong.
And it was not only in recently added feed, but everywhere on the app. Of course if the content was published few days ago, you wouldn't notice, so most of the time you would see this in recently added feed, but not only.
